### ISSUE TITLE
Link to blog in README is broken

### ISSUE THREAD

**Issue by robertmryan** [lorenbrichter/SimFinger#24]
Do you have a new link to your blog? Can I recommend that the README link to that blog be updated?

**Comment by arzdev** [comment]
Same issue link Is done :(

**Comment by robertmryan** [comment]
To give credit where credit is due, it was "Programmer" who brought this to my attention. :)

**Comment by arzdev** [comment]
Yes the link is down and I would like some new SimFinger Backgrounds. Thank
You.

**Comment by twe4ked** [comment]
This link “kind of” works: http://atebits.tumblr.com/post/197564349/not-your-average-iphone-screencast
